src/RamlToOpenApiConverter/RamlConverter.Paths.cs

```diff
@@ -30,8 +32,42 @@ private OpenApiPaths MapPaths(IDictionary<object, object> o)
             return paths;
         }
 
-        private ICollection<(OpenApiPathItem Item, string AdjustedPath)> MapPathItems(string parent, IList<OpenApiParameter> parentParameters, IDictionary<object, object> values)
+        private ICollection<(OpenApiPathItem Item, string AdjustedPath)> MapPathItems(string parent, IList<OpenApiParameter> parentParameters, IDictionary<object, object> values, IDictionary<object, object> uses)
         {
+            //replace uses in file
+            IDictionary<object, object> use_replace = new Dictionary<object, object>();
+
+            if (uses.Count > 0)
+            {
+                foreach (var path_value in values)
+                {
+
+                    var _is_val = ((Dictionary<object, object>)path_value.Value).GetAsString(Constants.Is_tag);
+
+                    if (_is_val != null)
+                    {
+                        var path_is_separator = _is_val.ToString().Split('.');
+                        foreach (var use in uses)
+                        {
+                            if (use.Key.ToString() == path_is_separator[0].ToString())
+                            {
+                                use_replace = ((Dictionary<object, object>)use.Value).GetAsDictionary(Constants.Root_uses);
+
+                                for (int i = 1; i < path_is_separator.Count(); i++)
+                                {
+                                    use_replace = use_replace.GetAsDictionary(path_is_separator[i]);
+                                }
+
+                                ((Dictionary<object, object>)path_value.Value).Replace(use_replace, Constants.Is_tag);
+
+                            }
+                        }
+

```

Comment:
remove this empty line

Reply:
https://github.com/StefH/RamlToOpenApiConverter/pull/18/commits/b71268a88f549db26c01754b3e2d645817418c21